third_party/istio-1.1.7/values-extras.yaml

```diff
@@ -0,0 +1,127 @@
+# Common settings.
+global:
+  omitSidecarInjectorConfigMap: true
+  istioNamespace: istio-system
+
+  proxy:
+    envoyStatsd:
+      # If enabled is set to true, host and port must also be provided. Istio no longer provides a statsd collector.
+      enabled: false
+      host: # example: statsd-svc.istio-system
+      port: # example: 9125
+
+
+gateways:
+  enabled: true
+
+  istio-ingressgateway:
+    enabled: false
+  istio-egressgateway:
+    enabled: false
+  istio-ilbgateway:
+    enabled: false
+
+  cluster-local-gateway:
+    enabled: true
+    labels:
+      app: cluster-local-gateway
+      istio: cluster-local-gateway
+    replicaCount: 1
+    autoscaleMin: 1
+    autoscaleMax: 1
+    resources: {}
+      # limits:
+      #  cpu: 100m
+      #  memory: 128Mi
+      #requests:
+      #  cpu: 1800m
+      #  memory: 256Mi
+    cpu:
+      targetAverageUtilization: 60
```

Comment:
Why does this differ from the other values files?

Reply:
I made a mistake. 60 is the value copied from Istio 1.1.3. 
Istio 1.1.7 and later use 80 as default value.